client/trackercyf/src/App.js

```diff
@@ -1,10 +1,24 @@
-import logo from './logo.svg';
-import './App.css';
+// Update 2 now also connect to trainee.js and mentor.js
+import React, { useState } from "react";
+import logo from "./logo.svg";
+import "./App.css";
+import MenuBar from "./MenuBar";
+import Trainee from "./trainee";
+import Mentor from "./mentor";
 
 function App() {
+  // Define a state variable to track the user's role
```

Comment:
i think it would be better to set the default role to a "trainee", because bull can cause issues for us in the future